exercises/02-publish-subscribe/after/Divergent.Finance/Host.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Threading.Tasks;
+using Divergent.Finance.PaymentClient;
+using Divergent.Sales.Messages.Events;
+using ITOps.EndpointConfig;
+using NServiceBus;
+using NServiceBus.Logging;
+
+namespace Divergent.Finance
+{
+    class Host
```

Comment:
This is almost a duplication from the Customers host.

Reply:
It is, and I considered putting most of `Host.cs`, `WindowsService.cs` and `Program.cs` into shared files, but it started to feel like "over-re-use". In any case, this is no worse than the duplication we currently have in master with `ContainerSetup.cs` and `EndpointConfig.cs`. In fact, I'd argue that there's less duplication in this branch. I'd prefer we could merge as-is and address the duplication (which is effectively pre-existing) in a separate PR, should we choose to.